vnext/Shared/FeatureGate.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <string>
+
+namespace Microsoft::React {
+
+void SetFeatureGate(std::string &&name) noexcept;
+
+bool GetFeatureGate(std::string &&name) noexcept;
```

Comment:
>bool [](start = 0, length = 4)

(nit) In future, we will want other types, eg int, but for now just having bool is ok.

Reply:
Agreed, but there is a chance the upgrade to 0.63 may make this mechanism obsolete.
Keeping it as scoped and small as possible for now.